app/queues/dns/workers/sync-db-status-worker.server.ts

```diff
@@ -5,21 +5,47 @@ import logger from '~/lib/logger.server';
 import { deleteDnsRecordById, updateDnsRecordStatusById } from '~/models/dns-record.server';
 import type { PollDnsStatusJobResult } from './poll-dns-status-worker.server';
 import { pollDnsStatusQueueName } from './poll-dns-status-worker.server';
+import { addNotification } from '~/queues/notifications/notifications.server';
 
 import type { DnsRecord } from '@prisma/client';
 import type { WorkType } from '../add-dns-record-flow.server';
+import { getUserByUsername } from '~/models/user.server';
 
 export const syncDbStatusQueueName = 'sync-db-status';
 
 export interface DbRecordSynchronizerData {
   workType: WorkType;
   id: DnsRecord['id'];
+  username: DnsRecord['username'];
 }
-
+const notifyUserBasedOnStatus = async (
+  username: DnsRecord['username'],
+  status: DnsRecordStatus
+) => {
+  // get user information from DB
+  let user = await getUserByUsername(username);
+  if (!user) {
+    throw new Error(`Could not get user by username: ${username}`);
+  }
+  switch (status) {
+    case DnsRecordStatus.active:
+      return addNotification({
+        emailAddress: user.email,
+        subject: 'Sample DNS Record active subject',
```

Comment:
Since I am getting all record data (including `status`) from the DB to send notifications, do I still need the `switch` statement? I remember it was mentioned that Route53 and DB could be out of sync.

Reply:
Actually, let me just test it out.